data/ships.txt

```diff
@@ -947,7 +951,8 @@ ship "Cruiser"
 	outfits
 		"Particle Cannon" 4
 		"Sidewinder Missile Launcher" 2
-		"Sidewinder Missile" 100
+		"Sidewinder Missile" 250
+		"Sidewinder Storage Rack" 6
```

Comment:
about the biggest ammo buff I made
if someone thinks it's a really big deal I can nerf it to 4, but it's a warship after all